### PR TITLE
Read account & org information from CSV

### PR DESCRIPTION
**Pull request by kahowell**
I considered separating CSV logic into a new class, but opted to simply
put it in the FileBasedOrgListStrategy, since the account number mapping
is a temporary solution.

For deployment we'll need to update our config file to a CSV instead of
a plain-text file.

### CODE REVIEW

File: src/main/java/org/candlepin/insights/controller/InventoryController.java [hunk 1]
```diff
@@ -180,9 +184,11 @@ private void extractHypervisorFacts(Consumer consumer, Map<String, String> pinhe
     }
 
     private List<ConduitFacts> getValidatedConsumers(String orgId) {
+        String accountNumber = orgListStrategy.getAccountNumberForOrg(orgId);
         List<ConduitFacts> conduitFactsForOrg = new LinkedList<>();
         for (Consumer consumer : pinheadService.getOrganizationConsumers(orgId)) {
             ConduitFacts facts = getFactsFromConsumer(consumer);
+            facts.setAccountNumber(accountNumber);
```
Comment (mstead) [inline src/main/java/org/candlepin/insights/controller/InventoryController.java]: This is not actually getting applied to the facts that are sent to the server. If you look on line 195 we are foolishly getting a new set of facts before adding to the conduitFactsForOrg list.
```java
            if (violations.isEmpty()) {
                conduitFactsForOrg.add(getFactsFromConsumer(consumer));
            }
```
And should be:
```java
            if (violations.isEmpty()) {
                conduitFactsForOrg.add(facts);
            }
```
Looks like an old bug and we should be adding the facts instead. :)

Comment (kahowell) [reply src/main/java/org/candlepin/insights/controller/InventoryController.java]: Good catch. Pushing update w/ a fix.

Comment (mstead) [reply src/main/java/org/candlepin/insights/controller/InventoryController.java]: Looks good here.

File: src/main/java/org/candlepin/insights/controller/InventoryController.java [hunk 2]
```diff
@@ -199,14 +205,20 @@ else if (log.isInfoEnabled()) {
     }
 
     public void updateInventoryForOrg(String orgId) {
-        List<ConduitFacts> conduitFactsForOrg = getValidatedConsumers(orgId);
+        String accountNumber = orgListStrategy.getAccountNumberForOrg(orgId);
+        Objects.requireNonNull(
+            accountNumber,
+            String.format("Org %s cannot be reported to inventory without an account number", orgId)
+        );
```
Comment (mstead) [inline src/main/java/org/candlepin/insights/controller/InventoryController.java]: Let's get a test around this.

**Pull request merged**